easybuild/easyconfigs/r/RSeQC/RSeQC-2.6.3-foss-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,42 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'PythonPackage'
+
+name = 'RSeQC'
+version = '2.6.3'
+
+homepage = 'http://rseqc.sourceforge.net/'
+description = """RSeQC provides a number of useful modules that can 
+ comprehensively evaluate high throughput sequence data especially RNA-seq 
+ data. Some basic modules quickly inspect sequence quality, nucleotide 
+ composition bias, PCR bias and GC bias, while RNA-seq specific modules 
+ evaluate sequencing saturation, mapped reads distribution, coverage 
+ uniformity, strand specificity, transcript level RNA integrity etc."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [SOURCEFORGE_SOURCE]
+
+python = 'Python'
+pyver = '2.7.10'
+versionsuffix = '-%s-%s' % (python, pyver)
+
+dependencies = [
+    ('R', '3.2.3'),
+    (python, pyver),
+    ('numpy', '1.10.1', versionsuffix),
```

Comment:
numpy is already provided through the `Python` dep?

is you need a newer numpy version than is provided by the Python dep, please mention that explicitly in a comment